realm/realm-library/src/main/java/io/realm/RealmCache.java

```diff
@@ -152,13 +156,18 @@ public void run() {
             } catch (InterruptedException e) {
                 RealmLog.warn(e, "`CreateRealmRunnable` has been interrupted.");
             } catch (final Throwable e) {
-                RealmLog.error(e, "`CreateRealmRunnable` failed.");
-                notifier.post(new Runnable() {
-                    @Override
-                    public void run() {
-                        callback.onError(e);
-                    }
-                });
+                // DownloadingRealmInterruptedException is treated specially.
```

Comment:
Because `DownloadingRealmInterruptedException` is a Sync exception, so it cannot be referenced directly from Base.